src/draw/lv_draw_vector.c

```diff
@@ -228,10 +228,10 @@ void lv_vector_path_get_bounding(const lv_vector_path_t * path, lv_area_t * area
         if(p[i].y > y2) y2 = p[i].y;
     }
 
-    area->x1 = (int32_t)x1;
-    area->y1 = (int32_t)y1;
-    area->x2 = (int32_t)x2;
-    area->y2 = (int32_t)y2;
+    area->x1 = (int32_t)roundf(x1);
```

Comment:
Use `lroundf` directly?

```suggestion
    area->x1 = lroundf(x1);
```

Reply:
Yes, you are right.